include/btu/modmanager/mod_folder.hpp

```diff
@@ -83,6 +83,7 @@ class ModFolder
     using enum ModFolderIteratorBase::ArchiveTooLargeState;
 
     explicit ModFolder(Path directory, bsa::Settings bsa_settings);
+    explicit ModFolder(Path directory, bsa::Settings bsa_settings, bool ignore_existing_archives);
```

Comment:
You do not have to add a constructor, you can default the parameter to false. I think that's what you did at first?

Reply:
Good point, changed it.